docs/source/algorithms/SumOverlappingTubes-v1.rst

```diff
@@ -44,6 +44,28 @@ Instrument Parameters
 
 If the instrument parameter ``mirror_detector_angles`` is set to true, then output scattering angles are effectively multiplied by :math:`-1`. This is used for D2B where the instrument is mirrored compared with other Powder Diffraction instruments at the ILL.
 
+SplitCounts
++++++++++++
+
+The counts are split between bins, if the difference between to angles is larger than the scattering angle step times the ``ScatteringAngleTolerance``.
```

Comment:
`to` should be `two`. I would also say `...scattering angle step multiplied by the...`.